mne/epochs.py

```diff
@@ -3288,6 +3288,12 @@ def make_fixed_length_epochs(raw, duration=1.,
     duration : float
         Duration of each epoch in seconds. Defaults to 1.
     %(preload)s
+    reject_by_annotation : bool
+        Whether to reject based on annotations. If ``True`` (default), epochs
+        overlapping with segments whose description begins with ``'bad'`` are
+        rejected. If ``False``, no rejection based on annotations is performed.
```

Comment:
can you factorize this docstring using docdict to avoid a copy paste?

Reply:
A lot of functions duplicate the documentation of `reject_by_annotation`. I updated most of them too (except those default to `False`).